### PR TITLE
Admin page done

### PR DESCRIPTION
**Pull request by sbagh**
Admin page done with css

also:
- added some comments to actions.js back-end file

- changed variable name BikeCtrl to BikeControl in bike-router.js -

### CODE REVIEW

File: client/src/pages/admin/admin-page.jsx [hunk 1]
```diff
@@ -1,9 +1,62 @@
...
+   useEffect(() => {
+      getAllBikes()
+         .then((value) => value.data)
+         .then((val) => {
+            setBikes(val.data);
+         })
+         .finally();
+   }, [bikes]);
+
+   function reservationStatusLogic(bikeReservationStatus) {
```
Comment (Hasan17AA) [inline client/src/pages/admin/admin-page.jsx]: `const reservationStatusLogic = (bikeReservationStatus) => {`
please change it to ES6 
line 23 as well

File: client/src/pages/admin/admin-page.jsx [hunk 2]
```diff
@@ -1,9 +1,62 @@
...
+      if (!bikeReservationStatus) {
+         return "available";
+      } else {
+         return "bike reserved";
+      }
+   }
+
+   function displayReports(bikeReportsArray) {
+      if (bikeReportsArray.length === 0) {
```
Comment (Hasan17AA) [inline client/src/pages/admin/admin-page.jsx]: `if (!bikeReportsArray.length ) {`

File: client/src/pages/admin/admin-page.jsx (hunk 1 above)
Comment (sbagh) [reply client/src/pages/admin/admin-page.jsx]: done

File: client/src/pages/admin/admin-page.jsx (hunk 2 above)
Comment (sbagh) [reply client/src/pages/admin/admin-page.jsx]: done

**Pull request merged**